apis/doesthedogdie.py

```diff
@@ -4,10 +4,10 @@
 import time
 import json
 try:
-    from config import use_memcache
+    from .config import use_memcache
     if  use_memcache:
         try:
-            from config import memcache_address, memcache_port, invalidation_time
+            from .config import memcache_address, memcache_port, invalidation_time
```

Comment:
just noticed these changes were left in. I cannot remember off the top of my head if they were needed or not. I will test with and without them this evening.